src/commcare_cloud/terraform/modules/awsmq/main.tf

```diff
@@ -0,0 +1,40 @@
+data "aws_secretsmanager_secret" "AMQP_USER" {
+  name = "commcare-staging/AMQP_USER"
+}
+
+data "aws_secretsmanager_secret_version" "AMQP_USER" {
+  secret_id = data.aws_secretsmanager_secret.AMQP_USER.id
+}
+
+locals {
+  AMQP_USER = data.aws_secretsmanager_secret_version.AMQP_USER.secret_string
+}
+
+data "aws_secretsmanager_secret" "AMQP_PASSWORD" {
+  name = "commcare-staging/AMQP_PASSWORD"
```

Comment:
can this be more generic instead of commcare-staging ? as when we will got for india and production then this could be an issue